core-ui/src/components/Extensibility/useGetCRbyPath.js

```diff
@@ -1,15 +1,19 @@
 import { useMicrofrontendContext } from 'shared/contexts/MicrofrontendContext';
 
 export const useGetCRbyPath = () => {
-  const { customResources = [] } = useMicrofrontendContext();
-  const resource = customResources.find(el => {
+  const { customResources = [], namespaceId } = useMicrofrontendContext();
+
+  const resource = (customResources || []).find(el => {
```

Comment:
`|| []` is already at line 4
```suggestion
  const resource = (customResources).find(el => {
```